WordPress/src/main/res/values/strings.xml

```diff
@@ -2356,6 +2356,14 @@
 
     <string name="google_error_timeout">Google took too long to respond. You may need to wait until you have a stronger internet connection.</string>
 
+    <!-- Post-Signup Interstitial -->
+    <string name="post_signup_interstitial_title">Welcome to WordPress</string>
+    <string name="post_signup_interstitial_subtitle">Whatever you want to create or share, we’ll help you do it right here.</string>
+    <string name="post_signup_interstitial_create_new_site">@string/site_picker_create_wpcom</string>
+    <string name="post_signup_interstitial_add_self_hosted_site">@string/site_picker_add_self_hosted</string>
```

Comment:
We should mark these strings `translatable="false"`. Note: I'm fine using `site_picker_create_wpcom` and `site_picker_add_self_hosted` directly from the interstitial if we find more generic names.

Reply:
> We should mark these strings `translatable="false"`.

Done. I had those in initially but forgot to add them back after removing them. 🤦‍♂️ Good catch!

> Note: I'm fine using `site_picker_create_wpcom` and `site_picker_add_self_hosted` directly from the interstitial if we find more generic names.

I'm fine with that as well. I was just trying to follow the perceived convention here, but after talking to @theck13 and @malinajirka about it, I now know the convention is actually very loose.

Personally, I would prefer naming them like this:

```
<string name="welcome_to_wordpress_title">Welcome to WordPress</string>
<string name="welcome_to_wordpress_subtitle">Whatever you want to create or share, we’ll help you do it right here.</string>
```
or even
```
<string name="welcome_to_wordpress">Welcome to WordPress</string>
<string name="whatever_you_want_to_create_or_share">Whatever you want to create or share, we’ll help you do it right here.</string>
```

and then rename `site_picker_create_wpcom` and `site_picker_add_self_hosted` to something like `create_wpcom_site` and `add_self_hosted_site`. 

In previous projects, I also used some prefixes to help distinguish different contexts in a more generic way, like `label_`, `action_`, `message_`, `title_`, etc. But I wouldn't use these here.

That being said, I decided to keep the screen name prefixes and leave the existing strings unchanged so I could continue with the non-intrusive approach I've been following so far. As @malinajirka said, every approach here has its downsides and there's probably a larger conversation that needs to take place to discuss this further, so I'd rather leave it that way until then.

Reply:
Works for me.

> In previous projects, I also used some prefixes to help distinguish different contexts in a more generic way, like label_, action_, message_, title_, etc. But I wouldn't use these here.

We've been discussing having more formal string reference, it could also help with translations (like `action.verb.trash` is easier to translate than just `trash`), but we're not there yet.